src/routes/about/+page.svelte

```diff
@@ -0,0 +1,44 @@
+<h1>About Us</h1>
+
+<h2>History</h2>
+<p>
+	Kendo Club at the University of Michigan was founded in the fall of 2001 and officially recognized
+	by the Midwest Kendo (MWKF) in the fall of 2002. Instruction of our club was originally lead by
+	Chika and Takashi Ezaki. Once they returned to Japan in 2003, it transitioned to Yukio Watanabe
+	and Joe Ponchart. After the ground work was laid out for a consistent practice space at the
+	University of Michigan, cross training with the Kendo club at Eastern Michigan University
+	continued and still continues to this day. Our shared history with EMU through Eastern Kendo
+	reaffirms our continued partnership as both clubs seek to advance and promote the practice of
+	Kendo in the Midwest. Many of our instructors and members train at both locations even to this
+	day.
+</p>
+<p>
+	Our club is classified as a student organization through the Michigan Student Assembly, the
+	student government of the University of Michigan. Our Kendo Club is also part of the
+	<a href="https://www.midwestkendofederation.org/">Midwest Kendo Federation (MWKF)</a>
+	which is the regional chapter under the
+	<a href="https://www.auskf.org/">All United States Kendo Federation (AUSKF)</a>, the national
+	governing body for Kendo in this country. The MWKF is responsible for all Kendo clubs and Dojos in
+	the Midwest area.
+</p>
+
+<h2>Motto</h2>
+
+<p style="font-size:30px">心技一体</p>
+
+<p>
+	Our club motto reads "Shin Gi Ittai", which translates approximately to "The Spirit and Skill are
+	One Body" or "The Heart and Technique are Together". This motto was one of several proposed by
+	Masaki Hori in the fall of 2004, and was chosen by our club officers and coaches to best represent
+	the ideals we strive for in our club. This motto is on our club tenugui (cloth head wrap worn
+	under helmet), which are traditionally given to club members before their first competition as a
+	sign of mutual support and encouragement.
+</p>
+
+<h2>Constitution</h2>
+<p>
+	As required by the University of Michigan and the SOAR (Student Organization and Recognition)
+	Initiative, <a href="https://maizepages.umich.edu/organization/kendoclub"
+		>our club constitution can be found on Maize pages.
```

Comment:
I think it would make more sense for just "maze pages" to be in the anchor because the link goes to maze pages not to the constitution.

Reply:
Fixed in [c20a74a](https://github.com/kendoclub-at-umich/website/pull/33/commits/c20a74a26da81efd26b9dad40a641589c288435c)